packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginCardSubmitButton.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Icon } from '../../../../index';
+
+const LoginCardSubmitButton = ({ isDisabled, children, isLoading }) => (
+  <Button className="login-pf-submit-button" type="submit" bsStyle="primary" bsSize="large" block disabled={isDisabled}>
+    {children}
+    {isLoading ? (
+      <span>
+        {' '}
+        <Icon name="spinner" spin size="lg" />
```

Comment:
perhaps use the spinner component?

Reply:
Do we have one ? :O

Reply:
Here's a link to the [PF3 react spinner](https://944-pr-patternfly-react-patternfly.surge.sh/patternfly-3/index.html?knob-Loading=true&knob-Size=sm&selectedKind=patternfly-react%2FWidgets%2FSpinner&selectedStory=Spinner&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs) (using your storybook preview url, btw)